README.md

```diff
@@ -24,10 +24,7 @@ If it has been some time since you last worked on the E2E project it is recommen
 $ make clean
 ```
 
-This will remove all your local apps, clone them, check them out to the latest
-deployed-to-production branch. This has been a problem in the past due to stale
-branches being checked out and either being unbuildable or causing false test
-failures.
+This will remove all your local apps, clone them, check them out to the latest deployed-to-production branch (in case one of the apps is on and old branch).
```

Comment:
```suggestion
This will remove all your local apps, clone them and check them out to the `deployed-to-production` branch (in case one of the apps is on an old branch).
```

Reply:
Fixed in https://github.com/alphagov/publishing-e2e-tests/pull/467/commits/f669e11bc9071792a64981e6ed12047f4d765057.